boot/boot.go

```diff
@@ -473,3 +473,16 @@ func UpdateCommandLineForGadgetComponent(dev snap.Device, gadgetSnapOrDir string
 	}
 	return cmdlineChange, nil
 }
+
+// CompleteFactoryReset runs a series of steps in a run system that complete a
+// factory reset process.
+func CompleteFactoryReset(encrypted bool) error {
```

Comment:
MarkFactoryResetComplete would be more consistent with other names in this package